packages/mui-codemod/src/deprecations/tooltip-props/test-cases/actual.js

```diff
@@ -0,0 +1,24 @@
+import Tooltip from '@mui/material/Tooltip';
+
+<Tooltip
+  components={{ Arrow: ComponentsArrow }}
```

Comment:
Let's cover all slots (`arrow`, `popper`, `tooltip`, `transition`) on the test cases